custom_components/intex_spa/translations/de.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+	 "abort": {
+		"already_configured": "Ger�t ist bereits konfiguriert"
+	 },
+	 "error": {
+		"cannot_connect": "Verbindungsfehler",
+		"dns_not_known": "Fehler beim Aufl�sen der DNS-Adresse",
+		"unknown": "Unerwarteter Fehler"
+	 },
+	 "step": {
+		"user": {
+		  "description": "Wenn sie Hilfe bei der Einrichtung ben�tigen schauen sie hier: https://github.com/mathieu-mp/homeassistant-intex-spa",
+		  "data": {
+			 "name": "Ger�t Name",
+			 "host": "Ger�t-Adresse (IP)"
```

Comment:
This can either be an IP or a host[.domain], please remove the mention of the IP

Reply:
of course I can change it to UTF-8! oops...

regarding the mention of the IP - I found it a little obstruse and most people have the device on an internal IP. 
Maybe there is a better way of saying "fully qualified domain adress". (most people do not know what it means and have to look it up first)
I will try to come up with something better!